src/model/platform-setup/setup-mac.ts

```diff
@@ -30,13 +30,41 @@ class SetupMac {
     }
   }
 
+  private static getModuleParametersForTargetPlatform(targetPlatform: string): string {
+    let command = '';
+    switch (targetPlatform) {
+      case 'iOS':
+        command += '--module ios ';
+        break;
+      case 'tvOS':
+        command += '--module tvos ';
+        break;
+      case 'Android':
+        command += '--module android ';
+        break;
+      case 'MacOS':
+        command += '--module mac-il2cpp ';
+        break;
+      case 'WebGL':
+        command += '--module webgl ';
+        break;
+      default:
+        throw new Error(`Unsupported module for target platform: ${targetPlatform}.`);
```

Comment:
I think it's reasonable to add an explicit `unityModules` input which is a list of N modules and would be additive to the output of this case statement. and we only raise exception if the modules list empty.

e.g. as of 2020.2 some modules exist for for localization (e.g. `language-ja`, `language-ko`). These are complementary and completely unrelated to the targetPlatform

https://forum.unity.com/threads/package-ui-localization-is-available-in-2020-2.957173/